app/code/core/Mage/Adminhtml/Block/Widget/Grid.php

```diff
@@ -426,7 +423,7 @@ public function getColumnCount()
      * Retrieve grid column by column id
      *
      * @param   string $columnId
-     * @return  Varien_Object || false
+     * @return  Mage_Adminhtml_Block_Widget_Grid_Column|Varien_Object|false
```

Comment:
```suggestion
     * @return  Mage_Adminhtml_Block_Widget_Grid_Column|false
```
There are 2 instances of `$this->_columns[$columnId] =` in the class, each one is set to block `'adminhtml/widget_grid_column'`.